lib/l10n/app_fr.arb

```diff
@@ -255,285 +249,339 @@
-  "@addToMix": {},
-  "removeFromMix": "Enlever du mix",
+  "removeFromMix": "Retirer du Mix",
   "@removeFromMix": {},
-  "minutes": "Minutes",
-  "@minutes": {},
-  "redownloadedItems": "{count,plural, =0{Aucun re-téléchargement nécessaire.} =1{{count} élément re-téléchargé} other{{count} éléments re-téléchargés}}",
+  "addToMix": "Ajouter au Mix",
+  "@addToMix": {},
+  "redownloadedItems": "{count,plural, =0{Aucun re-téléchargement nécéssaire} =1{{count} élément re-téléchargé} other{{count} éléments re-téléchargés}}",
   "@redownloadedItems": {
     "placeholders": {
       "count": {
         "type": "int"
       }
     }
   },
-  "bufferDuration": "Durée du tampon",
+  "bufferDuration": "Taille du tampon",
   "@bufferDuration": {},
-  "bufferDurationSubtitle": "Combien de temps le lecteur doit mettre en mémoire tampon, en secondes. Nécessite un redémarrage.",
+  "bufferDurationSubtitle": "Taille du tampon du lecteur, en secondes. Changer ce paramètre nécessite un redémarrage.",
   "@bufferDurationSubtitle": {},
-  "disableGesture": "Désactiver les gestes",
-  "@disableGesture": {},
-  "removeFromPlaylistTooltip": "Retirer de la playlist",
-  "@removeFromPlaylistTooltip": {},
-  "removeFromPlaylistTitle": "Retirer de la playlist",
-  "@removeFromPlaylistTitle": {},
-  "removedFromPlaylist": "Retirer de la playlist.",
-  "@removedFromPlaylist": {},
-  "disableGestureSubtitle": "Option permettant de désactiver les gestes.",
-  "@disableGestureSubtitle": {},
-  "showUncensoredLogMessage": "Ce journal contient vos informations de connexion. L'afficher ?",
-  "@showUncensoredLogMessage": {},
-  "insertedIntoQueue": "Ajouter à la file d'attente.",
-  "@insertedIntoQueue": {
-    "description": "Snackbar message that shows when the user successfully inserts items into the play queue at a location that is not necessarily the end."
-  },
-  "refresh": "ACTUALISER",
-  "@refresh": {},
-  "confirm": "Confirmer",
-  "@confirm": {},
   "language": "Langue",
-  "@language": {},
-  "playNext": "Lire à la suite",
-  "@playNext": {
-    "description": "Popup menu item title for inserting an item into the play queue after the currently-playing item."
-  },
+  "confirm": "Confirmé",
+  "showUncensoredLogMessage": "Ce journal contient vos informations de connexion. Afficher tout de même ?",
   "resetTabs": "Réinitialiser les onglets",
-  "@resetTabs": {},
-  "noMusicLibrariesBody": "Finamp n'a trouvé aucune bibliothèque musicale. Assurez-vous que votre serveur Jellyfin contient au moins une médiathèque de type \"Musique\".",
-  "@noMusicLibrariesBody": {},
-  "noMusicLibrariesTitle": "Pas de bibliothèque musicale",
+  "noMusicLibrariesTitle": "Aucune bibliothèque musicale",
   "@noMusicLibrariesTitle": {
-    "description": "Title for message that shows on the views screen when no music libraries could be found."
+    "description": "Titre du message affiché sur l'écran de vues lorsqu'aucune bibliothèque musicale n'a été trouvée."
```

Comment:
lorsque aucune
(lorsqu' only works in front of pronouns, il elle on...)

Reply:
I've made those changes at night so...
Sorry for those basic errors !!

Reply:
No prob, don't take the very short comments as reproaches. After the first few, I saved some time and went straight to the point.
And again thanks, you did a great job !